File: IDAUtil/Model/DistributionListProperty.vb
Imports System.ComponentModel.DataAnnotations.Schema

Public Class DistributionListProperty
    <Column("[name]")>
    Public Property name As String
    <Column("[salesOrg]")>
    Public Property salesOrg As String
    <Column("[country]")>
    Public Property country As String
    <Column("[address]")>
    Public Property address As String
End Class

File: IDAUtil/Model/Properties/ServerProperty/DistributionListProperty.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace IDAUtil.Model.Properties.ServerProperty {
    public class DistributionListProperty {
        [Column("[name]")]
        public string name { get; set; }
        [Column("[salesOrg]")]
        public string salesOrg { get; set; }
        [Column("[country]")]
        public string country { get; set; }
        [Column("[address]")]
        public string address { get; set; }
    }
}